.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -100,6 +100,21 @@ body:
 
         Providing as much information as possible greatly helps us with reproducing the issues.
 
+  - type: dropdown
+    id: flags
+    attributes:
+      label: Feature flags
+      description: Are you using any Reanimated or React Native feature flags in your application?
+      options:
+        - 'Yes'
+        - 'No'
+
```

Comment:
How about adding another dropdown for "React Native release level" with options "stable", "experimental" and "canary"?